coalib/testing/GlobalBearTestHelper.py

```diff
@@ -0,0 +1,58 @@
+import os.path
+
+from queue import Queue
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.settings.Section import Section
+from coalib.testing.BaseTestHelper import BaseTestHelper
+
+
+def _get_test_path(test_dir,
+                   file):
+    return os.path.join(os.path.dirname(__file__),
```

Comment:
@NiklasMM, I am facing issue with this method. My requirement is the file path of that directory from where this method is called but obviously this written method returns `coalib/testing*` directory.